tests/diagnostic/job.py.in

```diff
@@ -34,37 +34,51 @@ vvv = {
 ph.MaxwellianFluidModel(
     bx=bx, by=bx, bz=bx,
     ex=bx, ey=bx, ez=bx,
-
     protons={"charge":-1, "density":density, **vvv},
     alpha={"charge":-1, "density":density, **vvv}
 )
 
-ph.FluidDiagnostics(
-    diag_type="density",            # choose in (rho_s, flux_s)
-    write_every=10,                 # write on disk every x iterations
-    compute_every=5,                # compute diagnostics every x iterations ( x <= write_every)
-    start_iteration=0,              # iteration at which diag is enabled
-    last_iteration=990,             # iteration at which diag is turned off
-    population_name="protons"       # name of the population for which the diagnostics is made
-  #,path = 'FluidDiagnostics1'      # where output files will be written, [default: name]
+ph.DiagnosticInfo(
```

Comment:
do we do: 
```
diag_options = {format:“phareh5”, options:{dir:“phare_ouputs/“}} ? (edited) 
```
in Simulation now or in another pr?

Reply:
it can be in this one